wapitiCore/main/wapiti.py

```diff
@@ -1057,7 +1057,10 @@ def wapiti_main():
                 raise InvalidOptionValue("-a", args.credentials)
 
         if "auth_type" in args:
-            wap.set_auth_type(args.auth_type)
+            if args.auth_type == "post" and args.starting_urls != []:
+                wap.crawler.auth_url = args.starting_urls[0]
```

Comment:
Nice idea to use `-s` :) We will have to document it though

Reply:
Thanks :) I will prepare a PR to update the documentation